src/sql/workbench/parts/dashboard/browser/media/flexbox.css

```diff
@@ -0,0 +1,72 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+.fullsize {
```

Comment:
I'd also like that we made these styles more specific, but I understand if its outside the scope.